sources/packages/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -133,11 +138,27 @@ export class ApplicationStudentsController extends BaseController {
   })
   @ApiBadRequestResponse({ description: "Form validation failed." })
   @ApiNotFoundResponse({ description: "Application not found." })
+  @ApiForbiddenResponse({
+    description: "You have a restriction on your account.",
+  })
   async submitApplication(
     @Body() payload: SaveApplicationDto,
     @Param("applicationId") applicationId: number,
-    @UserToken() userToken: IUserToken,
+    @UserToken() studentToken: StudentUserToken,
   ): Promise<void> {
+    try {
+      await this.applicationControllerService.offeringIntensityRestrictionCheck(
+        studentToken.studentId,
+        payload.data.howWillYouBeAttendingTheProgram,
```

Comment:
Not part of your PR @ann-aot.
IMO, we should be changing the long components name in the form and change it accordingly in our models for the student application. We should take it as a story and change the names. In this case howWillYouBeAttendingTheProgram should be changed to offeringIntensity. Suggestions or thoughts @dheepak-aot @andrewsignori-aot @henryoforeh-aot @abschwenker

Reply:
We can have a ticket to handle this @guru-aot, there are some that are way bigger than this one.